### PR TITLE
Add recipe for export2graphlan

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: recipes/export2graphlan/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+about:
+    home: 'https://bitbucket.org/CibioCM/export2graphlan/overview'
+    summary: "export2graphlan is a conversion software tool for producing both annotation and tree file for GraPhlAn"
+package:
+  name: export2graphlan
+  version: '0.1.0'
```
Comment (johanneskoester) [inline recipes/export2graphlan/meta.yaml]: Can you set the version to the one defined [here](https://bitbucket.org/CibioCM/export2graphlan/src/d6c10c11ed4c5a6f2bf930c053b3b12e5473298e/export2graphlan.py?fileviewer=file-view-default)?

File: recipes/export2graphlan/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+build:
+  number: 0
+  skip: False
+source:
+  hg_url: https://hg@bitbucket.org/CibioCM/export2graphlan
+  hg_tag: tip
```
Comment (johanneskoester) [inline recipes/export2graphlan/meta.yaml]: hg_tag could be `d6c10c1` to ensure reproducibility.

**Comment by bebatut** [comment]
@johanneskoester  I fixed version and hg_tag as you suggested.
I will merge it if it is ok for you

File: recipes/export2graphlan/meta.yaml [hunk 3]
```diff
@@ -20,5 +20,11 @@ requirements:
       - scipy
       - numpy
 test:
+  requires:
```
Comment (bgruening) [inline recipes/export2graphlan/meta.yaml]: Why is this needed? It shouldn't be afaik.

Comment (johanneskoester) [inline recipes/export2graphlan/meta.yaml]: Indeed, the package itself with all its deps is always installed for the test.

Comment (bebatut) [inline recipes/export2graphlan/meta.yaml]: I finally removed them. I got an error during `docker` test, but it was due to lack of `numpy` in global requirements. So, I thought I had to had also the requirements for tests. But, it works (at least locally) without them

Comment (bgruening) [inline recipes/export2graphlan/meta.yaml]: Lets see what travis is saying. The test requirements are used for example to depend on `nose`, so testing specific dependencies.
